js/main.js

```diff
@@ -48,10 +79,14 @@ function entryDOM(entry) {
   $createImg.setAttribute('class', 'column-half');
   $createDivRow.appendChild($createDivColHalf);
   $createDivColHalf.setAttribute('class', 'column-half');
-  $createDivColHalf.appendChild($createH2);
+  $createDivColHalf.appendChild($justifyDiv);
+  $justifyDiv.setAttribute('class', 'justify-space row')
+  $justifyDiv.appendChild($createH2);
   $createH2.textContent = entry.title;
+  $justifyDiv.appendChild($createIcon);
   $createDivColHalf.appendChild($createP);
   $createP.textContent = entry.notes;
+
   $ul.appendChild($createLi);
```

Comment:
You should not be doing this inside this function, you should be appending the `return` where the function was called

Reply:
This still needs to be addressed